lib/src/utils/settings.dart

```diff
@@ -1,2 +1,2 @@
 // Agora AppId
-const APP_ID = '';
+const APP_ID = '764e530fde9542b1be2d2c67558a08c2';
```

Comment:
please avoid uploading your own appid

Reply:
Oh so sorry gonna remove this.. i accidentally uploaded this